frontend/src/layout/TopBar/TopBar.scss

```diff
@@ -12,6 +12,14 @@
     padding: 0;
 }
 
+.invalid-form:focus {
+    color: #e4e4e4;
+    background-color: #666666;
```

Comment:
I don't think background-color and color need to be there since they are inherited from the original form. Also please use variables (variables.scss) for defining colors. Other than that, good work 👍